### ISSUE TITLE
[BITV] 9.1.1.1a/2.1 - The Nextcloud logo (link) does not have an alternative text. A suitable alternative text could be "Nextcloud - Go to dashboard."

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36753]
- [ ] Fix
- [ ] Backport

![2b66d051c0e7a904424fb940d9032598](https://user-images.githubusercontent.com/1315170/219386835-211b0c58-f7ba-425b-be2b-8314c97de6f4.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v2-n1

</details>

**Comment by JuliaKirschenheuter** [comment]
@michaelnissenbaum It is actually not easy to set a detailed name for the logo-link because a link for the logo is configurable in: https://github.com/nextcloud/server/blob/dde5c46a3eb7c6dcee47795590619ccd393577d2/config/config.sample.php#L224-L233
This means it depends on the config and on user access/rights which link is available for a current user. 

For now the implementation looks like this which always says "(name of the nextcloud instance) homepage", whatever the current homepage is: 

![Screenshot from 2023-02-28 14-35-23](https://user-images.githubusercontent.com/6078378/221869657-51690d1f-92db-4d39-ba6c-c80dc798d4b2.png)

Could we move on with this solution? 
Thank you!

**Comment by skjnldsv** [comment]
The logo is irrelevant here. It should be treated as a decorative image.
What is important is the aria description of the link. You can edit this and have access to the `defaultapp` config with php.
Then use it to inject the proper aria attributes in the user template
https://github.com/nextcloud/server/blob/0d67fc23f4077e7b06a01bc519957f3f13d95f10/core/templates/layout.user.php#L61

**Comment by skjnldsv** [comment]
Extract the logic from https://github.com/nextcloud/server/blob/71f698649f578db19a22457cb9d420fb62c10382/lib/private/URLGenerator.php#L302-L338

And create a method to get the default AppId somewhere it make sense (Maybe [User.php](https://github.com/nextcloud/server/blob/ecad09b38a152957c771139562e61f7ecb1a6879/lib/private/User/User.php) ? :shrug: )
